include/copc-lib/hierarchy/box.hpp

```diff
@@ -0,0 +1,56 @@
+#ifndef COPCLIB_HIERARCHY_BOX_H_
```

Comment:
should we make a new folder for these you think? utils or geometry or something?